ckanext/scheming/plugins.py

```diff
@@ -22,8 +25,6 @@
     add_template_directory,
 )
 
-from paste.reloader import watch_file
```

Comment:
@smotornyuk this was the hook I was talking about on Tuesday. We need one of these on flask. Would be a good candidate for addition to ckantoolkit

Reply:
I haven't yet a chance to check werkzeug's observers, but it seems that they are not exposing any handles to the outside world.

While `paste` has this `watch_file`, that internally call function, that reloads server, `Flask` requires the complete list of watched files that are registered during server startup. If it's true than having something like `watch_file` inside `ckantoolkit` can cause a lot of problems - it must be called only on the initialization stage, not in runtime, but people definitely will call it everywhere. In that case, it would be safer to use only paths\globs specified statically in the config file and don't introduce any programmatical way to control server reloading.

Theoretically, we can patch `config` object and touch any source file\config file - thus Flask's observer will trigger reload, thinking that config\code was modified, and will fetch all the changes, but it's too tricky solution for keeping it in code.

Reply:
PS I hadn't completely removed this watcher - there is an attempt to use it in on L380. So, existing CKAN versions still support reloading and we still have time to include solution into the upcoming version